.github/labeler.yml

```diff
@@ -101,6 +101,8 @@ SQL:
     ]
     - any-glob-to-any-file: [
      'common/unsafe/**/*',
+     'common/sketch/**/*',
```

Comment:
May I ask why this is `SQL` module?

Reply:
Because the `classes` in it are all called by the SQL module, eg:
<img width="630" alt="image" src="https://github.com/apache/spark/assets/15246973/1f039e0d-e3e0-4c53-a861-add3738ff5bf">

<img width="781" alt="image" src="https://github.com/apache/spark/assets/15246973/6667351f-1a81-4eae-8edb-bd15e35bd5f4">

Reply:
Got it. It makes sense to me.